tests/app_client/cmd.py

```diff
@@ -118,6 +119,7 @@ def sign_tx(
             if sw != 0x9000:
                 raise DeviceException(error_code=sw, ins=InsType.INS_SIGN_TX)
 
+        time.sleep(0.1)
```

Comment:
Some tests would fail because speculos automation would be slower than pytest and we would send data before the device is ready.